config.json

```diff
@@ -4993,24 +5011,6 @@
         "type": "androidlibrary",
         "mavenRepositoryType": "MavenCentral"
       },
-      {
```

Comment:
Why are we removing these packages? They appear to still be maintained (latest release is Apr 1, 2025).

Reply:
They were duplicates and causing `duplicate types`

Those 2 artifacts are still in `config.json`

https://github.com/dotnet/android-libraries/blob/dev/moljac/wsu-20250322/config.json#L4996-L5013